dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -30,6 +30,7 @@ constexpr std::wstring_view expectedPushServerArgs = L"----WindowsAppRuntimePush
 
 static wil::unique_event g_waitHandleForArgs;
 static winrt::guid g_comServerClsid{ GUID_NULL };
+static bool registering{ false };
```

Comment:
Can this be a member variable instead?

Reply:
No I think it is fine let this be static.